modules/service/src/main/scala/lucuma/odb/util/GmosCodecs.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.util
+
+import lucuma.core.enums.GmosAmpGain
+import lucuma.core.enums.GmosAmpReadMode
+import lucuma.core.enums.GmosNorthFilter
+import lucuma.core.enums.GmosNorthFpu
+import lucuma.core.enums.GmosNorthGrating
+import lucuma.core.enums.GmosRoi
+import lucuma.core.enums.GmosSouthFilter
+import lucuma.core.enums.GmosSouthFpu
+import lucuma.core.enums.GmosSouthGrating
+import lucuma.core.enums.GmosXBinning
+import lucuma.core.enums.GmosYBinning
```

Comment:
I think eventually these need to go away, as they're defined in database tables and there is no way to keep them in sync. We can generate the GraphQL `enum` definitions at runtime and then pass them around in server code as type `lucuma.odb.data.Tag`, as I'm doing for partners now. Client code can fetch the metadata and construct `Enumerated` instances at runtime as we have discussed before. Maybe we should work on this when we're all in LSC.